src/client/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import './footer.css'
+
+export const Footer = () => {
+
```

Comment:
It will be great if you indent your code, makes it easier to read :)

Reply:
I will be carefully. Thanks